build/lib/generate_help_body.sh

```diff
@@ -80,7 +80,7 @@ if [[ "$HAS_PATCHES" == "true" ]]; then
 fi
 
 IMAGE_TARGETS_HELP=""
-if [ ! -z "$LOCAL_IMAGE_TARGETS" ]; then
+if [ ! -z "$(echo "$LOCAL_IMAGE_TARGETS" | xargs)" ]; then
```

Comment:
not sure what xargs would do here

Reply:
it trims the string of trailing spaces.. its a hack